packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -829,6 +829,17 @@
             </div>
         </div>
 
+        <div class="row d-md-flex">
+            <!-- Advices -->
+            <div class="col-md-4">
```

Comment:
As the `.col-md-4` seems to always be applied, I wonder if the `ovh-advices` component could set its own width

Reply:
In some cases, advices tile is shown just below another tile, in the same column. To handle this kind of requirement, its good to leave placement and width decisions for component users.